helm/resource_release.go

```diff
@@ -638,16 +638,12 @@ func resourceReleaseDelete(ctx context.Context, d *schema.ResourceData, meta int
 
 	name := d.Get("name").(string)
 
-	res, err := action.NewUninstall(actionConfig).Run(name)
+	_, err = action.NewUninstall(actionConfig).Run(name)
 
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	if res.Info != "" {
```

Comment:
I agree that we don't need to return this as an error here, but I would propose that we return a `Warning` diagnostic instead so the message still gets surfaced to the user. Something like this:

```go
diag.Diagnostic{
    Severity:  diag.Warning,
    Summary:   "Helm uninstall returned an information message",
    Detail:    res.Info,
}
```

Reply:
Thanks ! We knew we needed to keep retiring something. We just didn't know what exactly the terraform sdk allowed us to use. We will update quickly and update the Pr. 

Thansk for your insight.

Reply:
Updated. Please review. @jrhouston